widget/gridlayout.go

```diff
@@ -234,7 +234,14 @@ func (g *GridLayout) rowStretched(r int) bool {
 }
 
 func (g *GridLayout) preferredColumnWidthsAndRowHeights(widgets []PreferredSizeLocateableWidget) ([]int, []int) {
-	colWidths := make([]int, g.columns)
+	clen := g.columns
```

Comment:
Can we get a better name for this variable?

- columnCount
- colCount
- colLen
- colLength 

Something like that? At first I thought it was a misspelling of "clean"